scripts/experiment_1/functions/cca_core.py

```diff
@@ -0,0 +1,342 @@
+# Copyright 2016 Google Inc.
```

Comment:
Is it really necessary to keep copy pasting this script? Why not just create a bin where all external scripts are located and call from it? You seem to have at least two copies of this script.

Reply:
Very good point.  I have modified the directories accordingly